resources/views/components/input/shortcode-textarea.blade.php

```diff
@@ -0,0 +1,53 @@
+@props([
+    'name' => 'body',
+    'enabled' => true,
+    'watermark' => '',
```

Comment:
nit: I recommend renaming this to `placeholder`.

`enabled` is fair so as to not create a double negative on the standard `disabled` attribute, but I perceive less value in renaming `placeholder`.

Reply:
The component was extracted from https://github.com/RetroAchievements/RAWeb/blob/d74d235417dcfd4ca26e9fbba744ce5746a3c6b4/public/viewtopic.php#L213-L254 and I just chose names for each customizable bit of data.

I've changed `watermark` to `placeholder` as suggested.